.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+### Description
+
+Briefly describe the issue. Feel free to cross-reference your report with a
+related GitHub or [JIRA][jira] issue for additional context.
+
+### Environment
+
+What operating system and PHP version are you using? Mention if PHP was
+installed through a third-party bundle (e.g. MAMP, XAMPP).
+
+Include version information for the extension, libmongoc, and libbson. This may
+be collected from `phpinfo()`:
+
+```
+$ php -i | grep -E 'mongodb|libmongoc|libbson'
+```
+
+Before reporting an issue, check that you are using the most recent version of
+the [extension][pecl].
+
+### Test Script
+
+If possible, include a PHP script that reproduces the issue. Clarify whether
+this script can be run from the CLI or if it can only be reproduced through a
+web SAPI.
```

Comment:
For web-specific issues, request that they reproduce the bug/crash using PHP's built-in web server and provide a one-line demonstrating how to execute it.